rust_icu_sys/build.rs

```diff
@@ -22,7 +22,7 @@
 #[cfg(feature = "use-bindgen")]
 mod inner {
     use {
-        anyhow::{Context, Result},
+        anyhow::{anyhow, Context, Ok, Result},
```

Comment:
This does not seem to compile.

```
  Compiling rust_icu_sys v4.1.0 (/src/rust_icu/rust_icu_sys)
error[E0432]: unresolved import `anyhow::Ok`
  --> rust_icu_sys/build.rs:25:35
   |
25 |         anyhow::{anyhow, Context, Ok, Result},
   |                                   ^^ no `Ok` in the root
   |
   = help: consider importing one of these items instead:
           std::result::Result::Ok
           core::result::Result::Ok

For more information about this error, try `rustc --explain E0432`.
error: could not compile `rust_icu_sys` (build script) due to previous error
```

Not sure how it compiled in CI

Reply:
I see - so `anyhow` is resolved to version `1.0.45` on my machine, and that edition does not have `Ok`.

Reply:
`cargo update` to the rescue.

Reply:
Yes, feel free to pin to a specific minor version number.